stable/external-dns/templates/deployment.yaml

```diff
@@ -21,6 +23,10 @@ spec:
       {{- end }}
       {{- if .Values.priorityClassName }}
       priorityClassName: "{{ .Values.priorityClassName }}"
+      {{- end }}
+      {{- if .Values.podSecurityContext }}
+      securityContext:
```

Comment:
https://github.com/helm/charts/pull/14082 just merged - can you match the style and use nindent and keep toYaml in the same line?

Reply:
Done.